samples/python/classification_sample_async/classification_sample_async.py

```diff
@@ -109,7 +109,7 @@ def main() -> int:
 
 # --------------------------- Step 5. Loading model to the device -----------------------------------------------------
     log.info('Loading the model to the plugin')
-    compiled_model = core.compile_model(model, args.device)
+    compiled_model = ov.compile_model(model, args.device)
```

Comment:
looks like we need to revert changes in samples, because they are updated here https://github.com/openvinotoolkit/openvino/commit/f9c0e9690a03977ce22d956defda0cf0fbf76434